server/test/suite/fireboltOpenRpc.test.mjs

```diff
@@ -0,0 +1,42 @@
+"use strict";
+import { jest } from "@jest/globals";
+import * as fireboltOpenRpc from "../../src/fireboltOpenRpc.mjs";
+
+test("fireboltOpenRpc.getRawMeta working properly", () => {
+  const expectedResult = {};
+  const result = fireboltOpenRpc.getRawMeta();
+  expect(result).toEqual(expectedResult);
```

Comment:
How does this work? Doesn't seem like it should...

Reply:
In this fireboltOpenRpc.mjs file, getRawMeta function is returning an object which can't be accessible from test files as its blank object initially, that's why its returning an empty object and we are just checking with deep equality.
Please confirm.